ee/acl/acl_integration_test.go

```diff
@@ -0,0 +1,444 @@
+//go:build integration
+
+/*
+ * Copyright 2023 Dgraph Labs, Inc. and Contributors
+ *
+ * Licensed under the Dgraph Community License (the "License"); you
+ * may not use this file except in compliance with the License. You
+ * may obtain a copy of the License at
+ *
+ *     https://github.com/dgraph-io/dgraph/blob/main/licenses/DCL.txt
+ */
+
+package acl
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/dgraph-io/dgo/v210/protos/api"
+	"github.com/dgraph-io/dgraph/dgraphtest"
+	"github.com/dgraph-io/dgraph/x"
+)
+
+func (suite *AclTestSuite) TestInvalidGetUser() {
+	t := suite.T()
+	hc, err := suite.dc.HTTPClient()
+	require.NoError(t, err)
+	hc.HttpToken.AccessJwt = "invalid Token"
+	currentUser, err := hc.GetCurrentUser()
+	require.NoError(t, err)
+	require.Equal(t, "null", currentUser)
+	require.Equal(t, x.GqlErrorList{{
+		Message: "couldn't rewrite query getCurrentUser because unable to parse jwt token: token" +
+			" contains an invalid number of segments",
+		Path: []interface{}{"getCurrentUser"},
+	}}, err)
+}
+func (suite *AclTestSuite) TestPasswordReturn() {
+	t := suite.T()
+	hc, err := suite.dc.HTTPClient()
+	require.NoError(t, err)
+	require.NoError(t, hc.LoginIntoNamespace(dgraphtest.DefaultUser,
+		dgraphtest.DefaultPassword, dgraphtest.GalaxyNamespace, ""))
+	query := `
+	query {
+		getCurrentUser {
+			name
+			password
+		}
+	}`
+
+	_, err = hc.RunGraphqlQuery(dgraphtest.GraphQLParams{Query: query}, true)
+	makeRequestAndRefreshTokenIfNecessary(t, dgraphtest.GraphQLParams{Query: query}, hc)
+	require.Equal(t, err, x.GqlErrorList{{
+		Message: `Cannot query field "password" on type "User".`,
+		Locations: []x.Location{{
+			Line:   5,
+			Column: 4,
+		}},
+	}})
+}
+
+func (suite *AclTestSuite) TestHealthForAcl() {
+	t := suite.T()
+	hc, err := suite.dc.HTTPClient()
+	require.NoError(t, err)
+	require.NoError(t, hc.LoginIntoNamespace(dgraphtest.DefaultUser,
+		dgraphtest.DefaultPassword, dgraphtest.GalaxyNamespace, ""))
+
+	params := dgraphtest.GraphQLParams{
```

Comment:
should we make a function to do this too?

Reply:
done